api/db/changelog/db.changelog-103-data-dictionary.xml

```diff
@@ -0,0 +1,43 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<databaseChangeLog
+  xmlns="http://www.liquibase.org/xml/ns/dbchangelog/1.9"
+  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog/1.9
+                      http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-1.9.xsd">
+  <changeSet author="ericsong" id="changelog-103-data-dictionary">
+
+    <createTable tableName="data_dictionary_entry">
+      <column name="data_dictionary_entry_id" type="bigint" autoIncrement="true">
+        <constraints primaryKey="true" nullable="false" />
+      </column>
+
+      <column name="cdr_version_id" type="bigint">
+        <constraints nullable="false" foreignKeyName="fk_data_dictionary_entry_cdr_version" references="cdr_version(cdr_version_id)"/>
+      </column>
+
+      <column name="defined_time" type="datetime">
+        <constraints nullable="false"/>
+      </column>
+
+      <column name="relevant_omop_table" type="varchar(255)">
+        <constraints nullable="false"/>
+      </column>
+
+      <column name="field_name" type="varchar(255)">
+        <constraints nullable="false"/>
+      </column>
+
+      <column name="omop_cdm_standard_or_custom_field" type="text"/>
+      <column name="description" type="text"/>
+      <column name="field_type" type="text"/>
+      <column name="data_provenance" type="text"/>
+      <column name="source_ppi_module" type="text"/>
+      <column name="transformed_by_registered_tier_privacy_methods" type="boolean"/>
+    </createTable>
+
+    <addUniqueConstraint
+      tableName="data_dictionary_entry"
+      columnNames="relevant_omop_table, field_name, cdr_version_id"
```

Comment:
I like this invariant, but is it respected by the input data?

Reply:
It sounds like it should be based off of my conversations with Karthik and Lauren. I can make a note to verify this when the input data comes in.